src/frontend/citizen-portal/src/app/api/api/trafficCourtsCitizenService.service.ts

```diff
@@ -0,0 +1,162 @@
+/**
+ * Traffic Court Online Citizen Api
+ * An API for creating violation ticket disputes
+ *
+ * The version of the OpenAPI document: v1
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+/* tslint:disable:no-unused-variable member-ordering */
+
+import { Inject, Injectable, Optional }                      from '@angular/core';
+import { HttpClient, HttpHeaders, HttpParams,
+         HttpResponse, HttpEvent, HttpParameterCodec, HttpContext 
+        }       from '@angular/common/http';
+import { CustomHttpParameterCodec }                          from '../encoder';
+import { Observable }                                        from 'rxjs';
+
+// @ts-ignore
+import { Invoice } from '../model/invoice.model';
+
+// @ts-ignore
+import { BASE_PATH, COLLECTION_FORMATS }                     from '../variables';
+import { Configuration }                                     from '../configuration';
+
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TrafficCourtsCitizenServiceService {
```

Comment:
I know it is generated code but why is it be "ServiceService", others do not have this problem. Is that the naming of backend?

Reply:
@KDNeufeld and invoice's part is not inside this PR, right ? Only because this PR needed to re-generate api files so invoice being included ?

Reply:
That's correct. That looks to be something that Phil added recently to the API but the citizen client hasn't been updated yet.

Reply:
Ok I will leave it as is for now